Sources/GoodExtensions/UIKit/UIViewExtensions.swift

```diff
@@ -12,12 +12,16 @@ import GRCompatible
 
 // MARK: - Initialization from XIB
 
-public extension GRActive where Base: UIView {
-
+public extension UIView {
+    
     static func loadFromNib() -> Self {
```

Comment:
Aky bol dovod toto zmenit? Aky je scenar pouzitia?

Reply:
Nooo, Vpodstate ty potrebujes vraciat Self a cez ten GRActive by si vratil GRActive a cez base by si vratil UIView ale stale to nieje self cize to bolo nepouzitelne lebo by si musel castovat na vyzadovany typ po pouziti metody. Neviem celkom ako by sa to dalo zapisat aby to vracialo Class.Self cez GRActive kedze ho nutis takto chovat sa ako GRActive a nie ako Class.Self a cez Base ho zasa nutis spadat pod UIView.